src/services/auditLogService.ts

```diff
@@ -0,0 +1,43 @@
+import * as gqlTypes from '../typeDefs/gqlTypes.js'
+import * as dbSchema from '../typeDefs/dbSchema.js'
+// import { ErrorCode, Result } from '../result.js'
+import { dbInstance } from '../firebaseDb.js'
+import { logger } from '../logger.js'
+
+type CreatedAuditLog = {
+    isSuccesful: boolean
```

Comment:
Should I create this type in the schema so it generates it in gqlType File?

Reply:
theoretical answer:
your db types should be independent from your api as some fields/tables/entites are not exposed to the client. Audit logs are a good example of something you don't want the user to really know about. To this point, these types should be created either in the service or dbtypes for ones persisted. 

practical answer:
we have an admin panel that will be able to see audit logs, so I think we can add it to the gql schema and use it from there.